src/scenes/CreateEvent/components/CreateEventCreatorPicker/index.js

```diff
@@ -26,10 +26,14 @@ class CreateEventCreatorPicker extends React.PureComponent {
     lastUsedAddress: PropTypes.string.isRequired,
     setLastUsedAddress: PropTypes.func.isRequired,
     changeFormFieldValue: PropTypes.func.isRequired,
-    eventEscrowAmount: PropTypes.number.isRequired,
+    eventEscrowAmount: PropTypes.number,
     intl: intlShape.isRequired, // eslint-disable-line react/no-typos
   };
 
+  static defaultProps = {
+    eventEscrowAmount: 100,
```

Comment:
let's revert back to the required since 100 is not accurate as the default value